new-lamassu-admin/src/components/photo/index.js

```diff
@@ -0,0 +1,3 @@
+import Photo from './Photo'
```

Comment:
I don't see this used anywhere, nor anything under this photo component. Also remove .DS_Store from the commit

Reply:
This was an idea that I abandoned but forgot to delete.